microsoft/botframework-solutions
# [TypeScript][Skill Sample] Add manifest endpoint

## Description
- Add `/api/skill/manifest` endpoint to the bot.
- Update template with the new endpoint.

## Testing Steps
1. Go to `.\templates\Virtual-Assistant-Template\typescript\samples\sample-skill` and open a PowerShell terminal.
1. Run `npm install` to install the dependencies.
1. Run the `.\deployment\scripts\deploy.ps1` script with the needed args.
1. Run `npm start` to run the bot.
1. Go to `http://localhost:3980/api/skill/manifest`.
1. Check that the new sample shows the manifest file.

![image](https://user-images.githubusercontent.com/12394167/59133005-0cb22780-894d-11e9-86a1-2d1066ae9d0a.png)
